src/com/goide/settings/GoPluginSettings.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2013-2015 Sergey Ignatov, Alexander Zolotov, Mihai Toader, Florin Patan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.goide.settings;
+
+import com.goide.GoConstants;
+import com.goide.util.GoUtil;
+import com.intellij.openapi.components.*;
+import com.intellij.openapi.roots.impl.ModuleRootManagerComponent;
+import org.jdom.Element;
+import org.jetbrains.annotations.Nullable;
+
+@State(
+  name = GoConstants.GO_SERVICE_NAME,
+  reloadable = true,
+  storages = {
+    @Storage(id = "other", file = StoragePathMacros.APP_CONFIG + "/" + GoConstants.GO_CONFIG_FILE, roamingType = RoamingType.PER_PLATFORM)
+  }
+)
+public class GoPluginSettings implements PersistentStateComponent<Element> {
```

Comment:
You can just use existing PropertiesComponent for flags like projectTutorialShown. No need create new service for that.